File: javascript/src/models/integrationKeyOut.ts
// this file is @generated

export interface IntegrationKeyOut {
  key: string;
}

export const IntegrationKeyOutSerializer = {
  _fromJsonObject(object: any): IntegrationKeyOut {
    return {
      key: object["key"],
    };
  },

  _toJsonObject(self: IntegrationKeyOut): any {
    return {
      key: self.key,
    };
  },
};

File: rust/src/models/integration_key_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct IntegrationKeyOut {
    pub key: String,
}

impl IntegrationKeyOut {
    pub fn new(key: String) -> Self {
        Self { key }
    }
}
